### PR TITLE
adds profile

### PR DESCRIPTION
**Pull request by pheeque**
pheeque hng

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -58,11 +58,14 @@
...
           require __DIR__ . '/views/hackthievist.php';
           break;
+      case '/interns/pheeque' :
+          require __DIR__ . '/views/interns/pheeque.php'; 
+          break;
       case '/interns/mercyikpe' :
           require __DIR__ . '/views/interns/mercyikpe.php';
       case '/interns/aniebiet' :
           require __DIR__ . '/views/interns/aniebiet.php';
-          break;
```
Comment (Victor-Ugwueze) [inline index.php]: Why did you remove this line?

Comment (pheeque) [reply index.php]: i did not. the other two cases did not have break statements. git made it look like i removed it when it actually wasn't present at the time i added my commit.
